### ISSUE TITLE
Translation doesn't work

### ISSUE THREAD

**Issue by Beau-Yang** [tsujan/Kvantum#607]
Hello, I'm using Manjaro KDE and my locate is zh_CN.UTF-8. 
I tried to install the kvantum by the commond line:
```bash
sudo pacman -S kvantum-qt5
```
Howerver, I find that kvantummanager and kvantumpreview not showing with zh_CN translation. I have tried to uninstall the kvantum-qt5 and install it by the commond line:
```bash
yay -S kvantum-qt5-git
```
but it doesn't work.

I konw that the Chinese translation file is the kvantummanager_zh_CN.ts, but I only find the kvantummanager_zh_CN.qm under the location of /usr/share/kvantummanager/translations/. I notice that there's anther issue about the zh_CN translation not showing but it seems to require compiling Kvantum from its source. Could you please offer me some suggestions to make the translation works normally after installing Kvantum directly? Thanks in advance.

System info:
OS: Manjaro 20.2.1 Nibia
Kernel: x86_64 Linux 5.9.16-1-MANJARO

Locale:
LANG=zh_CN.UTF-8
LC_CTYPE="zh_CN.UTF-8"
LC_NUMERIC="zh_CN.UTF-8"
LC_TIME="zh_CN.UTF-8"
LC_COLLATE="zh_CN.UTF-8"
LC_MONETARY="zh_CN.UTF-8"
LC_MESSAGES="zh_CN.UTF-8"
LC_PAPER="zh_CN.UTF-8"
LC_NAME="zh_CN.UTF-8"
LC_ADDRESS="zh_CN.UTF-8"
LC_TELEPHONE="zh_CN.UTF-8"
LC_MEASUREMENT="zh_CN.UTF-8"
LC_IDENTIFICATION="zh_CN.UTF-8"
LC_ALL=

![image](https://user-images.githubusercontent.com/49222826/107873425-7b4ed300-6eed-11eb-842a-cacd6eee0065.png)

![image](https://user-images.githubusercontent.com/49222826/107873439-8e61a300-6eed-11eb-87c1-42d05f42169f.png)

**Comment by tsujan** [comment]
> Could you please offer me some suggestions to make the translation works normally after installing Kvantum directly?

All translations work by default — whether they are correct/complete or not is translators' responsibility.

When I use `LANG=zh_CN.UTF-8 kvantummanager` here (*exactly as your locale*), I get a translated GUI:

![zh_CN](https://user-images.githubusercontent.com/981076/107877176-0d4fdf00-6ee0-11eb-8d13-1957e82e4b6f.png)

You should have `qt5-translations` (which is needed for localization of Qt apps). If you have it, Manjaro's package may have a problem.

A direct compilation always gives correct translation files. If your GUI isn't translated after a direct compilation, I'm afraid the problem should be somewhere else (perhaps in  KDE).

**Comment by Beau-Yang** [comment]
I install the package you mention and the translation works. I'm new to the linux and not familiar with the Qt. Sorry to bother you about that and thanks a lot.

**Comment by tsujan** [comment]
> I install the package you mention and the translation works.

You mean `qt5-translations`?

> I'm new to the linux and not familiar with the Q

That's OK. All of us were new to Linux at some point and understand that.

**Comment by Beau-Yang** [comment]
Yes. I find that I haven't installed it before so I install it by the command line
```bash
sudo pacman -S qt5-translations
``
Then reopen the kvantum manager.

**Comment by tsujan** [comment]
Yes, it's needed for localization of all Qt apps — perhaps except for KDE apps because KDE used its independent translation system with old versions of Qt (< 4) and has kept it until today.

**Comment by tsujan** [comment]
By the way:

I checked Manjaro's `kvantum-qt5`. As a matter of fact, it has a small issue. It should include `qt5-translations` as an optional dependency, so that the user can know what he may miss; but it doesn't.

**Comment by ArchangeGabriel** [comment]
It has been decided long ago that the optional dependency was to be on [`qt5-base`](https://archlinux.org/packages/extra/x86_64/qt5-base/), where you can see ` qt5-translations (optional) - for some native UI translations`, instead of on each package requiring it.

But like all transitive optional dependencies (for another example, GTK apps can display numerous media formats with the correct dependencies installed –librsvg,libheif,libavif,webp-pixbuf-loader…– but they are optional for gdk-pixbuf2 instead of the apps), this is poorly documented in my regard.

**Comment by tsujan** [comment]
> It has been decided long ago that the optional dependency was to be on qt5-base

I didn't know that. Yes, it's the best choice.

**Comment by ArchangeGabriel** [comment]
> > It has been decided long ago that the optional dependency was to be on qt5-base
> 
> I didn't know that. Yes, it's the best choice.

Technically speaking yes, but from a UX perspective it lacks discoverability. People will rarely install or look at `qt5-base` directly, and if they miss the message about available optdeps on its first install, they will never know about it (it won’t be displayed upon installation of other apps using it and eventually being able to gain more from having those optdeps installed).

**Comment by tsujan** [comment]
Users may ignore optional dependencies — or "Recommends", as they're called in Debian — but if they do so, they'll be responsible for the lack of optional features, some of which can be quite important. My mistake, in my comment on Feb 14, was that I thought `qt5-translations` wasn't introduced as an optional dependency anywhere.

I installed Manjaro Testing 5 years ago and I still use it — with continuous upgrades, of course,. I clearly remember that it notified me of translations and I installed them.
